vendor/CppQuickCheck-2018-03-28/include/cppqc/Generator.h

```diff
@@ -364,7 +365,11 @@ void sampleShrinkOutput(const Generator<T> &g, std::ostream &out,
             T x = g.unGen(rng, i);
             std::vector<T> shr = g.shrink(x);
             if (randomized)
-                std::random_shuffle(shr.begin(), shr.end());
+            {
```

Comment:
I was wondering about this one, but it seems (per cppreference) that this overload should have been removed in C++17, but apparently the compilers aren't always as strict as they should. TIL.